app/views/admin/export.html.haml

```diff
@@ -0,0 +1,93 @@
+                %i.large.material-icons.circle.blue.lighten-2 description
+                %span.title People
+                %p
+                  = link_to(content_tag(:i, 'system_update_alt', class: 'material-icons left') + 'Download',
+                    admin_person_export_path(format: 'csv'),
+                    class: 'waves-effect waves-light btn download-button')
+
+              %li.collection-item.avatar{:style => "min-height:65px"}
+                %i.large.material-icons.circle.blue.lighten-2 description
+                %span.title ProtocolSubscriptions
+                %p
+                  = link_to(content_tag(:i, 'system_update_alt', class: 'material-icons left') + 'Download',
+                    admin_protocol_subscription_export_path(format: 'csv'),
+                    class: 'waves-effect waves-light btn download-button')
+
+              %li.collection-item.avatar{:style => "min-height:65px"}
+                %i.large.material-icons.circle.blue.lighten-2 description
+                %span.title InvitationSets
+                %p
+                  = link_to(content_tag(:i, 'system_update_alt', class: 'material-icons left') + 'Download',
+                    admin_invitation_set_export_path(format: 'csv'),
+                    class: 'waves-effect waves-light btn download-button')
+
+              %li.collection-item.avatar{:style => "min-height:65px"}
+                %i.large.material-icons.circle.blue.lighten-2 description
+                %span.title ProtocolTransfers
+                %p
+                  = link_to(content_tag(:i, 'system_update_alt', class: 'material-icons left') + 'Download',
+                    admin_protocol_transfer_export_path(format: 'csv'),
+                    class: 'waves-effect waves-light btn download-button')
+
+              %li.collection-item.avatar{:style => "min-height:65px"}
+                %i.large.material-icons.circle.blue.lighten-2 description
+                %span.title Rewards
+                %p
+                  = link_to(content_tag(:i, 'system_update_alt', class: 'material-icons left') + 'Download',
+                    admin_reward_export_path(format: 'csv'),
+                    class: 'waves-effect waves-light btn download-button')
+
+              %li.collection-item.avatar{:style => "min-height:65px"}
+                %i.large.material-icons.circle.blue.lighten-2 description
+                %span.title Proof of participation
+                %p
+                  = link_to(content_tag(:i, 'system_update_alt', class: 'material-icons left') + 'Download',
+                    admin_proof_of_participation_export_path(format: 'csv'),
+                    class: 'waves-effect waves-light btn download-button')
+
+              - @pilot_questionnaires = Questionnaire.pilot
+              - @rest_questionnaires = Questionnaire.all - @pilot_questionnaires
+              - @rest_questionnaires.each do |questionnaire|
```

Comment:
Misschien een methode `export` definieren in de admin_controller waar we deze variabelen aanmaken? OF: deze code wordt gedeeld met de `preview_overview` methode, dus daar kan je ook een gezamelijke before_action van maken en dan staat het nog maar op 1 plek.